python/tests/conftest.py

```diff
@@ -117,6 +125,38 @@ def moto_s3_uri_incl_bucket(moto_s3_endpoint_and_credentials):
     ] + ":" + bucket + "?access=" + aws_access_key + "&secret=" + aws_secret_key + "&port=" + port
 
 
+@pytest.fixture
+def moto_azure_endpoint_and_credentials(azurite_port, spawn_azurite):
+    global BUCKET_ID
+
+    container = f"testbucket{BUCKET_ID}"
+    BUCKET_ID = BUCKET_ID + 1
+
+    credential_name = "devstoreaccount1"
+    credential_key = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
+    endpoint = "0.0.0.0:" + str(azurite_port)
+    is_https = False
+    connect_to_azurite = True
+    yield endpoint, container, credential_name, credential_key, is_https, connect_to_azurite
```

Comment:
I think you can use `return` here.

Reply:
I don't think this is resolved!

You only need to use `yield` in a fixture if you need to do things with the fixture post-test. If you just want to return a fixture value, use `return`.

Reply:
I understand. I thought I have resolved it already. It is on me sorry.